### ISSUE TITLE
overriding footnotes

### ISSUE THREAD

**Issue by dmliao** [probablyup/markdown-to-jsx#240]
Hi there,

Would it be possible to use `options` to override the HTML representations of footnotes (without also changing all other links)? 

I'd like to be able to replace each footnote with an inline tooltip, sort of like how [bigfoot](http://www.bigfootjs.com/) renders them. That might require exposing footnotes separately in options, since their DOM representation isn't a 1 to 1 HTML tag.

**Comment by probablyup** [comment]
Need to think about this - what you actually need is the footnote data available if you pass an alternate anchor tag component as an override.

**Comment by erwagasore** [comment]
@probablyup yeah, with the option to override footnote and make available the data it will give a user more room to be creative on how they display their footnotes. I would love to have this feature 👍🏾
